eclair-core/src/main/scala/fr/acinq/eclair/json/JsonSerializers.scala

```diff
@@ -399,22 +399,15 @@ object OriginSerializer extends MinimalSerializer({
   })
 })
 
-object GlobalBalanceSerializer extends MinimalSerializer({
-  case o: GlobalBalance =>
-    val formats = DefaultFormats + ByteVector32KeySerializer + BtcSerializer + SatoshiSerializer
-    JObject(JField("total", JDecimal(o.total.toDouble))) merge Extraction.decompose(o)(formats)
-})
-
 // @formatter:off
+private case class GlobalBalanceJson(total: Double, onChain: CorrectedOnChainBalance, offChain: OffChainBalance)
+object GlobalBalanceSerializer extends ConvertClassSerializer[GlobalBalance](b => GlobalBalanceJson(b.total.toDouble, b.onChain, b.offChain))
```

Comment:
```suggestion
private case class GlobalBalanceJson(total: Btc, onChain: CorrectedOnChainBalance, offChain: OffChainBalance)
object GlobalBalanceSerializer extends ConvertClassSerializer[GlobalBalance](b => GlobalBalanceJson(b.total, b.onChain, b.offChain))
```

Reply:
Done in https://github.com/ACINQ/eclair/pull/2104/commits/e28956e4dacc3e630b88c41660074e5e6361568d